TVRemoteControl/ScannerTableViewController.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  ScannerTableViewController.swift
+//  TVRemoteControl
+//
+//  Created by Daian Aiziatov on 26/02/2019.
+//  Copyright © 2019 robot64. All rights reserved.
+//
+
+import UIKit
+
+class ScannerTableViewController: UITableViewController {
+
+    private lazy var progressView: UIProgressView = {
+        let frames = CGRect(x: 0,
+                            y: (self.navigationController?.navigationBar.frame.height ?? 0) + 20,
+                            width: (self.navigationController?.navigationBar.frame.width ?? 0),
+                            height: 20)
+        let progressView = UIProgressView(frame: frames)
+        progressView.translatesAutoresizingMaskIntoConstraints = false
+        return progressView
+    }()
+
+    private var connectedTVs = [SystemInfo]()
+
+    private lazy var scanner: LANScanner = {
+        return LANScanner()
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
```

Comment:
Do you remember what we discussed about `MVC` approach with working with `View`? Let's arrange a workshop where we perform refactoring of this screen.

Reply:
I am down to workshop. Agreed, that this controller is poorly designed, but it is just for testing 🐷 Although It might be reused 🤔